Jenkinsfile

```diff
@@ -20,134 +20,170 @@ pipeline {
+                  sh "sudo mv registries.conf /etc/containers/registries.conf"
+                  sh "sudo curl https://raw.githubusercontent.com/kubernetes-sigs/cri-o/$CRIO_VERSION/test/policy.json -o /etc/containers/policy.json"
+                  sh "sudo curl -L https://github.com/opencontainers/runc/releases/download/$RUNC_VERSION/runc.amd64 -o /usr/bin/runc"
+                  sh "sudo chmod +x /usr/bin/runc"
+                }
+              }
             }
           }
         }
-        stage("make vet") {
-          steps {
-            dir(path: "$REPO_DIR") {
-              sh "make vet"
+        stage("make vet, lint, cyclomatic"){
+          parallel {
+            stage("make lint") {
+              steps {
+                dir(path: "$REPO_DIR") {
+                  sh "make lint"
+                }
+              }
             }
-          }
-        }
-        stage("make cyclomatic-check") {
-          steps {
-            dir(path: "$REPO_DIR") {
-              sh "make cyclomatic-check"
+            stage("make format") {
+              steps {
+                dir(path: "$REPO_DIR") {
+                  sh "make format"
+                }
+              }
+            }
+            stage("make vet") {
+              steps {
+                dir(path: "$REPO_DIR") {
+                  sh "make vet"
+                }
+              }
+            }
+            stage("make cyclomatic-check") {
+              steps {
+                dir(path: "$REPO_DIR") {
+                  sh "make cyclomatic-check"
+                }
+              }
+            }
+            stage("make test BUILDTAGS=kerneldrv") {
+              steps {
+                dir(path: "$REPO_DIR") {
+                  sh "make test BUILDTAGS=kerneldrv"
+                }
+              }
             }
           }
         }
-        stage("make test BUILDTAGS=kerneldrv") {
+        stage('make pre-pull') {
           steps {
-            dir(path: "$REPO_DIR") {
-              sh "make test BUILDTAGS=kerneldrv"
+    	      dir(path: "$REPO_DIR") {
+              sh "make pre-pull"
             }
           }
         }
-      }
-    }
-    stage('make pre-pull') {
-      steps {
-	dir(path: "$REPO_DIR") {
-          sh "make pre-pull"
-        }
-      }
-    }
-    stage('make images') {
-      parallel {
-        stage("make images with docker") {
-          steps {
-            dir(path: "$REPO_DIR") {
-              sh "make -j4 images"
+        stage('make images') {
+          parallel {
+            stage("make images with docker") {
+              steps {
+                dir(path: "$REPO_DIR") {
+                  sh "make -j4 images"
+                }
+              }
+            }
+            stage("make images with buildah") {
+              steps {
+                dir(path: "$REPO_DIR") {
+                  sh "make images BUILDER=buildah"
+                }
+              }
```

Comment:
lots of whitespaces?

Reply:
I'been all day trying to see why github mark those whitespaces, if there are spaces, seems that previous commits had some not detected whitespaces: https://github.com/intel/intel-device-plugins-for-kubernetes/pull/231/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3L103.  

I been rewriting this file line by line and there are still. :/
I think could be a good time to start using a convention to write a Jenkinsfile.
My suggest is to start using grovy-language format, a sample of the result Jenkinfile can be seen in this alternative uri: https://github.com/juliocri/intel-device-plugins-for-kubernetes/pull/1/files